socorro/unittest/cron/jobs/test_monitoring.py

```diff
@@ -253,15 +252,11 @@ def test_run_log(self, app_config):
             description='This is an error',
         )
 
-        mocks = [
-            mock.patch.object(app, 'get_python_vulnerabilities', return_value=[vuln]),
-            mock.patch.object(app, 'get_javascript_vulnerabilities', return_value=[]),
-            mock.patch.object(app, 'alert_log'),
-        ]
-
-        with nested(*mocks):
-            app.run()
-            app.alert_log.assert_called_with([vuln])
+        with mock.patch.object(app, 'get_python_vulnerabilities', return_value=[vuln]):
+            with mock.patch.object(app, 'get_javascript_vulnerabilities', return_value=[]):
+                with mock.patch.object(app, 'alert_log'):
```

Comment:
`contextlib.nested` was deprecated because it has weird corner cases (I don't know what they are offhand). So, this just manually nests them.

There were a couple of instances of this, but that's it. I figured I'd just do this for now and if we have a better way of handling this, we can change it later.